resources/default_pipeline_environment.yml

```diff
@@ -395,6 +395,13 @@ steps:
         && mkdir -p \$GOPATH/src/${config.whitesource.projectName.substring(0, config.whitesource.projectName.lastIndexOf('/'))}
         && ln -s \$(pwd) \$GOPATH/src/${config.whitesource.projectName}
         && cd \$GOPATH/src/${config.whitesource.projectName} && dep ensure
+    dub:
+      buildDescriptorFile: './dub.json'
+      dockerImage: 'buildpack-deps:stretch-curl'
```

Comment:
@andre2007 does my proposal fit? Can you quickly test drive the setup?

Reply:
While every image (containing curl) fits, this is a sane decision. I saw the image installs gcc and g++. With GNU compiler collection version 9 ```D``` is included. When GCC 9 will be available from apt-get I will add ```gdc``` to the package list.

Reply:
@andre2007 THX for the feedback! So LGTM;-)